.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,7 @@
 
 ###### Things done
 
+- [ ] Added/updated the necessary docs
```

Comment:
I'd probably say "documentation" here, rather than abbreviating it as "docs". The "docs" abbreviation may not be immediately obvious to non-native English speakers.